src/lib/config.ts

```diff
@@ -1,7 +1,16 @@
 const env = process.env.NODE_ENV;
 const isDevelopment = env !== 'production';
 
+// added about to save the description somewhere
 const config = {
+  about: `With 100,000 members and 500+ chapters, the Association for Computing
+          Machinery is the world's largest society for computing. Here at UC
+          San Diego, our chapter has been established with the mission of
+          creating a member-first community devoted to the field of computing.
+          We welcome students of all backgrounds and skill levels to come
+          develop their skills at our many workshops and form new friendships at
+          our many socials. Get involved today by signing up for an event on
+          this portal or following us on social media!`,
```

Comment:
Since this description is only used on one page, it's better to keep it with the About page's files.